### ISSUE TITLE
Add haywire

### ISSUE THREAD

**Issue by waghanza** [the-benchmarker/web-frameworks#630]
+ Language : c
+ Repo : https://github.com/haywire/haywire
+ Author : @kellabyte

**Comment by waghanza** [comment]
thanks @kellabyte for given us consent on `irc`

This project compare performance based on 3 routes :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

**Comment by OvermindDL1** [comment]
Already have a libuv based test in the first C++ server so I'd imagine it would perform fairly identically, not too hard to make.  I can't at the moment (real life has me way busy) but if there is a bot or something to ping me here on occasion then I can look at it, doesn't seem hard, a bit wordy/verbose, but simple enough:  https://github.com/haywire/haywire/blob/master/src/samples/techempower_benchmark/program.c

**Comment by waghanza** [comment]
I'll ping you when I start a `PR`
